#!/usr/bin/env python
import math
import os.path
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("icr_x", double_t, 0, default=0.0,
        description="x coordinate (in m) of axis of axle's instant center of rotation (assuming ideal Ackermann"
                    " steering geometry); for typical steered vehicles, this is the x coordinate of the fixed axle")

package_name = os.path.basename(os.path.dirname(os.path.dirname(__file__)))
node_name = package_name + "_node"
cfg_name = os.path.splitext(os.path.basename(__file__))[0]
exit(gen.generate(package_name, node_name, cfg_name))
